ext/gd/gd.c

```diff
@@ -2738,6 +2783,16 @@ PHP_FUNCTION(imagegd2)
 }
 /* }}} */
 
+#ifdef HAVE_GD_BMP
+/* {{{ proto bool imagebmp(resource im [, string filename [, int compression]])
```

Comment:
Yeah originally about the "q != 0" part, I would probably have gone with something like "!!q" to let C decide on its boolean conversion value, but that should have no impact if ZPP is changed ;-)